applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -77,6 +62,59 @@ Make sure that the DFU lock utility is enabled if your nRF Desktop application c
 You cannot use this module with the :ref:`caf_ble_smp`.
 In other words, you cannot simultaneously enable the :ref:`CONFIG_DESKTOP_DFU_MCUMGR_ENABLE <config_desktop_app_options>` option and the :kconfig:option:`CONFIG_CAF_BLE_SMP` Kconfig option.
 
+You have to choose backend for the DFU process by enabling one of the following options:
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_MCUBOOT` - This option depends on the MCUboot bootloader support for the DFU process.
+    For details, see :ref:`mcuboot` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_SUIT` - This option depends on the SUIT tool support for the DFU process.
+    For details, see :ref:`ug_nrf54h20_suit_dfu` in the Zephyr documentation.
+
+.. note::
+    Currently b0 backend is not supported.
```

Comment:
I think that it is self explanatory looking at the options above and not needed here.
Also in this form IMO it kinda promises that it will be supported one day.

Reply:
@MarekPieta  asked me to add it here.

Reply:
if you both will insist on leaving it here then i would remove the "currently" part of this sentence and leave just: "B0 bootloader backend is not supported".

Reply:
Maybe: `B0 bootloader is not integrated with MCUmgr` (to avoid confusing it with lack of support in the application module)?

Reply:
sure, sounds fine